src/modding/NewModGUI.cs

```diff
@@ -217,6 +300,57 @@ private bool ReadControlsToEditedInfo()
         editedInfo.ModAssembly = modAssembly.Text;
         editedInfo.AssemblyModClass = assemblyModClass.Text;
 
+        if (!string.IsNullOrWhiteSpace(previewImagesFile.Text))
+        {
+            var previewImagesList = new List<string>();
+            Array.ForEach(previewImagesFile.Text.Split(","), s => previewImagesList.Add(s.Trim()));
+            editedInfo.PreviewImages = previewImagesList;
+        }
+
+        if (!string.IsNullOrWhiteSpace(dependencies.Text))
+        {
+            var dependenciesList = new List<string>();
+            Array.ForEach(dependencies.Text.Split(","), s => dependenciesList.Add(s.Trim()));
+            editedInfo.Dependencies = dependenciesList;
+        }
+
+        if (!string.IsNullOrWhiteSpace(requiredMods.Text))
+        {
+            var requiredModsList = new List<string>();
+            Array.ForEach(requiredMods.Text.Split(","), s => requiredModsList.Add(s.Trim()));
+            editedInfo.RequiredMods = requiredModsList;
+        }
+
+        if (!string.IsNullOrWhiteSpace(loadBefore.Text))
+        {
+            var loadBeforeList = new List<string>();
+            Array.ForEach(loadBefore.Text.Split(","), s => loadBeforeList.Add(s.Trim()));
+            editedInfo.LoadBefore = loadBeforeList;
+        }
+
+        if (!string.IsNullOrWhiteSpace(loadAfter.Text))
+        {
+            var loadAfterList = new List<string>();
+            Array.ForEach(loadAfter.Text.Split(","), s => loadAfterList.Add(s.Trim()));
+            editedInfo.LoadAfter = loadAfterList;
+        }
+
+        if (!string.IsNullOrWhiteSpace(incompatibleMods.Text))
+        {
+            var incompatibleModsList = new List<string>();
+            Array.ForEach(incompatibleMods.Text.Split(","), s => incompatibleModsList.Add(s.Trim()));
+            editedInfo.IncompatibleMods = incompatibleModsList;
+        }
```

Comment:
These lines look fairly redundants and should probably be factored one way or another.